test/bigtest/tests/details-order-test.js

```diff
@@ -17,18 +17,23 @@ import LineEditPage from '../interactors/line-edit-page';
 import OpenOrderConfirmationModal from '../interactors/PurchaseOrder/open-order-confirmation-modal';
 import OpenOrderErrorModal from '../interactors/PurchaseOrder/open-order-error-modal';
 import { ERROR_CODES } from '../../../src/components/Utils/order';
-// import LinesLimitModal from '../interactors/lines-limit-modal';
+import LinesLimitModal from '../interactors/lines-limit-modal';
+import { ORDER_TYPE } from '../../../src/components/PurchaseOrder/PODetails/FieldOrderType';
 
 describe('OrderDetailsPage', () => {
   setupApplication();
   const lineEditPage = new LineEditPage();
   const orderDetailsPage = new OrderDetailsPage();
   const orderEditPage = new OrderEditPage();
-  // const linesLimitModal = new LinesLimitModal();
+  const linesLimitModal = new LinesLimitModal();
   let order = null;
+  let line = null;
```

Comment:
I think you need to move this `let` a bit below, somewhere near to assignment

Reply:
But it used twice in this file

Reply:
that's the thing what confusing me. I see declaration of this var and expect that it's assigned with value always. But I still have to do it inside my describe.